src/org/jgroups/protocols/FlowControl.java

```diff
@@ -46,7 +41,7 @@
      * a REPLENISHMENT request to the members from which we expect credits. A value <= 0 means to wait forever.
      */
     @Property(description="Max time (in ms) to block")
-    protected long max_block_time=5000;
+    protected long max_block_time=500;
```

Comment:
It was intentional, but I only added the commit after Bela had already integrated the pull request.